gapii/cc/android/installer.cpp

```diff
@@ -197,7 +198,19 @@ void Installer::install_gles() {
         const char* name = gapii::kGLESExports[i].mName;
         void* func_export = gapii::kGLESExports[i].mFunc;
         bool import_found = false;
-        for (int i = 0; i < NELEM(gDriverPaths); ++i) {
+        if (drivers[0] != nullptr) { // libhwgl.so
+            // Huawei implements all functions in this library with prefix,
+            // all GL functions in libGLES*.so are just trampolines to his.
```

Comment:
Does this also include EGL functions?

Reply:
I am not sure.  Should I find out?

Reply:
Not really important, I just thought you said it did.